tools/mllm-llm-benchmark/models/Llama.hpp

```
@@ -0,0 +1,147 @@
+#include <chrono>
+#include <string>
+
+#include "BenchmarkTemplate.hpp"
+
+#include <mllm/mllm.hpp>
+#include <mllm/models/llama/modeling_llama.hpp>
+#include <mllm/models/llama/configuration_llama.hpp>
+
+class Llama_Benchmark final : public BenchmarkTemplate {
+ public:
+    std::optional<KVCacheEstimateInfo> kvEstimateInfo() const override {
```

Clang (14.0.6)

[error] function 'kvEstimateInfo' should be marked [[nodiscard]]

(modernize-use-nodiscard,-warnings-as-errors)